rpcs3/Emu/NP/np_handler.cpp

```diff
@@ -174,20 +174,23 @@ std::string np_handler::ip_to_string(u32 ip_addr)
 
 void np_handler::string_to_npid(const char* str, SceNpId* npid)
 {
-	strncpy(npid->handle.data, str, sizeof(npid->handle.data));
+	strncpy(npid->handle.data, str, sizeof(npid->handle.data)-1);
```

Comment:
nit:
```suggestion
	strncpy(npid->handle.data, str, sizeof(npid->handle.data) - 1);
```
same for the other occurences